packages/plugins/graphql/server/services/builders/type.js

```diff
@@ -150,6 +150,12 @@ module.exports = context => {
     const type = attribute.multiple
       ? naming.getRelationResponseCollectionName(fileContentType)
       : naming.getEntityResponseName(fileContentType);
+    const resolverPath = `${naming.getTypeName(contentType)}.${attributeName}`;
```

Comment:
same question can you explain what this is and how it relates to sanitize ?

Reply:
It's not related to `sanitize` but to the "remove restricted relations" topic. Adding an auth check for the scope corresponding to the content type of the relationship allows us to perform permissions check on resolve. (and thus remove relations the user don't have access to)